src/scss/base/_fonts.scss

```diff
@@ -1,4 +1,5 @@
 @font-face {
+	font-display: swap;
```

Comment:
Setting `font-display: swap` seems to add a small increase to performance (lighthouse score is constantly higher with `font-display: swap` then without)

Should I leave in these lines or remove them so the default value `auto` is used?

Reply:
> Should I leave in these lines or remove them so the default value `auto` is used?

These font faces are for text fonts. As we discussed yesterday, it makes sense to have a value defined for this category but better to have designer's input on picking a value that provides a preferred user experience.

I think it's OK to leave `font-display: swap` in here for now. Hopefully at the check-in next Monday, we'll figure out a proper value with @lliskovoi.

Reply:
Okay then this PR is ready for review